README.md

```diff
@@ -1,26 +1,37 @@
-![Screenshot 2023-09-04 at 12 54 11 PM](https://github.com/eundersander/siro_hitl_unity_client/assets/6557808/5aac5731-a7eb-4f0d-876c-871b330f6677)
+# Habitat VR HITL Client
+
+This application is a VR client that can remotely control a human agent in Habitat, in a human-in-the-loop (HITL) evaluation context.
+
+See [this document](https://github.com/facebookresearch/habitat-lab/blob/main/examples/siro_sandbox/VR_HITL.md) for setting up the server.
```

Comment:
... for setting up the HITL server. This is a "thin client", with most application complexity residing on the server. However, the client can be tested alone (without the server) as a simple model viewer, and we recommend this as a sanity check before running the full client/server application. See local testing instructions below.